tests/integration/SdkTest_test.cpp

```diff
@@ -4684,6 +4684,61 @@ TEST_F(SdkTest, SdkGetBanners)
     ASSERT_TRUE(err == MegaError::API_OK || err == MegaError::API_ENOENT) << "Get banners failed (error: " << err << ")";
 }
 
+TEST_F(SdkTest, SdkBackupFolder)
+{
+    getAccountsForTest(1);
+    LOG_info << "___TEST BackupFolder___";
+
+    //
+    // make this work even before the remote API has support for My Backups folder
+
+    // create My Backups folder
+    fetchnodes(0, maxTimeout);
+    std::unique_ptr<MegaNode> myBkpsNode{ megaApi[0]->getNodeByPath("/My Backups") };
```

Comment:
The node relative to the `/My Backups` folder is stored in a user's attribute and it needs to be retrieved from the user's attribute, not by searching by folder name.
Note that there might be a user's attribute pointing to the node, but the node could have been deleted. So you will need to check not only that the user's attribute exists and contains a node's handle, but also that the node's handle exists.